web/app.php

```diff
@@ -1,24 +1,16 @@
 <?php
 
-use Symfony\Component\ClassLoader\ApcClassLoader;
 use Symfony\Component\HttpFoundation\Request;
 
-$loader = require_once __DIR__.'/../app/autoload.php';
-include_once __DIR__.'/../var/bootstrap.php.cache';
-
-// Enable APC for autoloading to improve performance.
-// You should change the ApcClassLoader first argument to a unique prefix
-// in order to prevent cache key conflicts with other applications
-// also using APC.
-
-$apcLoader = new ApcClassLoader('sm', $loader);
-$loader->unregister();
-$apcLoader->register(true);
-
-//require_once __DIR__.'/../app/AppCache.php';
+require __DIR__.'/../vendor/autoload.php';
+if (PHP_VERSION_ID < 70000) {
```

Comment:
ok, in our case we only use php > 70000, so you can delete the ifs and code that is for php 5